docs/source/intro/get_started.rst

```diff
@@ -0,0 +1,194 @@
+****************
+Getting Started
+****************
+
+Installation
+============
+
+There are numerous routes to setting up the full AiiDA environment,
+depending on your use case.
+Here we first provide the simplest approach for installation on your local computer.
+
+`Conda <https://docs.conda.io>`_  provides a cross-platform package management system,
```

Comment:
While, until you arrived, I've been the guy in the AiiDA team pushing most for supporting the conda route, I still feel that before we make conda the default installation path we need a little more testing of this for production.
I guess, so far, you are the only one using the aiida-core coda package (? we should ask!).

Since the availability with python 3.6-3.8 has now been taken care of, my main remaining doubts are these:

 1. In my experience, people are easily confused when mixing `pip` and `conda` installations of packages and the will need to use pip to install most of the plugins. How can we effectively mitigate this?
 1. Do people prefer to have Postgresql & rabbitmq be a system service (that restarts when they restart their computer) or are they fine starting them manually every time?

I think we should discuss these two questions within the AiiDA team and, possibly, also in the `plugin-developers` github team.

And, if in the end we decide to go with conda, I'd be happy to integrate your aiida-active script in to the `aiida.services` package (or add a perhaps slightly reduced version of it as a dependency).

Reply:
Yes certainly this requires a group decision. The main thing with Conda is that it works cross-platform, so in this "quick install" I don't have to complicate things with discussing the OS dependent installations of the services. Obviously there will also be the advanced/detailed installation section, where I can go in to more detail about the different routes to setting up the services.

 > In my experience, people are easily confused when mixing pip and conda installations of packages

What would you say exactly is the confusion? Do we just need to say that you can use `pip` within a conda environment?

Reply:
> What would you say exactly is the confusion? Do we just need to say that you can use pip within a conda environment?

I've often heard of problems with dependencies being overwritten by pip / conda when doing this.

Reply:
> This can be very nasty, up to the point where people no longer know how to fix the problem.

Well just start again from a fresh environment 😉 Personally I have never had this problem, and the other thing is that it is very easy with Conda for people to share `conda-env.yaml` (which as you know can include conda and pip packages) containing version specifications that are known to be compatible.
But either way we should definitely be encouraging the use of isolated environments: e.g. https://medium.com/@krishnaregmi/pipenv-vs-virtualenv-vs-conda-environment-3dde3f6869ed